pyproject.toml

```
@@ -120,3 +121,14 @@ testfixtures = "*"
 tuspy = "*"
 twill = "*"
 watchdog = "*"
+
+[tool.black]
+line-length = 120
+target-version = ['py37']
+include = '\.pyi?$'
+extend-exclude = '''
+/(
+  | packages
+  | tools
+)/
+'''
```

```suggestion
'''
force-exclude = 'lib/galaxy/util/jstree.py'
```

We also ignore this file for flake8 in `setup.cfg` . Changes to `lib/galaxy/util/jstree.py` should also be reverted, I think.